File: Chapter08/SpringSessionDemo/src/main/java/com/dxtd/SpringSessionDemo/RedisHttpSessionConfiguration.java
package com.dxtd.SpringSessionDemo;

import org.springframework.context.annotation.Configuration;
import org.springframework.session.data.redis.config.annotation.web.http.EnableRedisHttpSession;

@Configuration
//maxInactiveIntervalInSeconds 默认是 1800 秒过期，这里测试修改为 120 秒
@EnableRedisHttpSession(maxInactiveIntervalInSeconds=120)
public class RedisHttpSessionConfiguration {

}

File: Chapter08/SpringSessionDemo/src/main/java/com/dxtd/SpringSessionDemo/RedisHttpSessionConfiguration.kt
package com.dxtd.SpringSessionDemo

import org.springframework.context.annotation.Configuration
import org.springframework.session.data.redis.config.annotation.web.http.EnableRedisHttpSession

/**
 * 开启Redis Http Session
 */
@Configuration
@EnableRedisHttpSession
public class RedisHttpSessionConfiguration {


}